services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -444,6 +444,12 @@ module.exports = {
       return isArray(sectionQuery) ? sectionQuery : [];
     },
 
+
+    magazineSchedules: async (content, _, { basedb }) => {
+      const contentMagazineSchedules = await basedb.find('magazine.Schedule', { 'content.$id': content._id });
+      return contentMagazineSchedules;
+    },
```

Comment:
```suggestion
    magazineSchedules: ({ _id }, _, { basedb }) => basedb.find('magazine.Schedule', { 'content.$id': _id }),
```

Just to clean this up a bit: There's no need to define a variable just to return it, and no need to make this an async function when only awaiting one thing. Just return the promise directly